README.md

```diff
@@ -35,6 +35,91 @@ This flow chart illustrates the conversion process provided by `convert.py`. It
 
 The initial script is in place for the Verilog conversion steps.
 
+## Installation and Setup
+
+### Prerequisites
+
+- Python 3.10 or higher
+- Git
+- Build tools (gcc, make, etc.)
+- OpenAI API key (for LLM integration)
+
+### System Dependencies
+
+The project requires several system-level tools for formal verification and synthesis:
+
+- Yosys (for synthesis)
+- SymbiYosys (for formal verification)
+- EQY (for equivalence checking)
+- Z3 (SMT solver)
+
+### Quick Start
+
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/stevehoover/conversion-to-TLV.git
+   cd conversion-to-TLV
+   ```
+
+2. Run the initialization script to install all dependencies:
+   ```bash
+   chmod +x init
```

Comment:
If it's not executable already, you can make it executable and commit it with those permissions.